### PR TITLE
Add `tinty config` subcommand to add config related information

### PR DESCRIPTION
**Pull request by JamyGolden**
`tinty config` with the relevant flags `--config-path` and `--data-path` can be useful when debugging to make sure that the expected config is applied and the expected config and data directories are being used by Tinty.

https://github.com/tinted-theming/tinty/issues/21

### CODE REVIEW

**Comment by laenzlinger** [comment]
looks good to me

maybe a `--config-file` could be even more useful for a user to directly edit the file

```
vi $(tinty config --config-file)
```

File: README.md [hunk 1]
```diff
@@ -346,6 +352,13 @@ hook = "cp -f %f ~/.config/sway/config && swaymsg reload"
   So the hook does a copy of the selected theme and replaces
   `~/.config/alacritty/colors.toml`.
 
+## Debugging
```
Comment (laenzlinger) [inline README.md:355]: Debugging sounds a bit misleading to me, but I also can not think of a better word

**Comment by JamyGolden** [comment]
There are sets of functionality with this:

- `tinty config` - prints the config.yml file
- `tinty config --config-path` - prints the path to config.yml, so you could `vi $(tinty config --config-path)`
- `tinty config --data-path` - prints the path to the data dir, which on linux is `~/.local/share/tinted-theming/tinty`

File: README.md (hunk 1 above)
Comment (JamyGolden) [reply README.md:355]: :+1:  I'll add some extra information to try to make it less ambiguous.

**Comment by JamyGolden** [comment]
Yes, I think I'll change `--data-path` to `--data-dir-path` to contrast with `--config-path` to remove ambiguity of `--config-path` returning a directory. Thanks for bringing that up!

**Pull request merged**